library/core/src/char/methods.rs

```diff
@@ -7,6 +7,9 @@ use crate::unicode::{self, conversions};
 
 use super::*;
 
+/// If 6th bit set ascii is upper case.
+const ASCII_CASE_MASK: u8 = 0b10_0000u8;
+
```

Comment:
Should I mention the trivia that this also happens to be the same bit patten as ascii space?

Reply:
Would probably not help :)